apps/api/src/server/app/case-officer/__tests__/case-officer.get-appeals.test.js

```diff
@@ -0,0 +1,200 @@
+// eslint-disable-next-line import/no-unresolved
+import test from 'ava';
+import supertest from 'supertest';
+import sinon from 'sinon';
+import { app } from '../../../app.js';
+import DatabaseFactory from '../../repositories/database.js';
+
+const request = supertest(app);
+
+const appeal_1 = {
+	id: 1,
+	reference: 'APP/Q9999/D/21/1345264',
+	appealStatus: [{
+		status: 'awaiting_lpa_questionnaire',
+		valid: true
+	}],
+	createdAt: new Date(2022, 1, 23),
+	addressId: 1,
+	localPlanningDepartment: 'Maidstone Borough Council',
+	planningApplicationReference: '48269/APP/2021/1482',
+	appellant: {
+		name: 'Lee Thornton'
+	},
+	startedAt: new Date(2022, 4, 18),
+	address: {
+		addressLine1: '96 The Avenue',
+		addressLine2: 'Maidstone',
+		postcode: 'MD21 5XY',
+		county: 'Kent'
+	}
+};
+const appeal_2 = {
+	id: 2,
+	reference: 'APP/Q9999/D/21/5463281',
+	appealStatus: [{
+		status: 'received_lpa_questionnaire',
+		valid: true
+	}],
+	createdAt: new Date(2022, 1, 25),
+	addressId: 2,
+	startedAt: new Date(2022, 4, 22),
+	address: {
+		addressLine1: '55 Butcher Street',
+		postcode: 'S63 0RB',
+		town: 'Thurnscoe'
+	}
+};
+const appeal_3 = {
+	id: 3,
+	reference: 'APP/Q9999/D/21/5463281',
+	appealStatus: [{
+		status: 'overdue_lpa_questionnaire',
+		valid: true
+	}],
+	createdAt: new Date(2022, 1, 25),
+	addressId: 2,
+	startedAt: new Date(2022, 4, 22),
+	address: {
+		addressLine1: '55 Butcher Street',
+		postcode: 'S63 0RB',
+		town: 'Thurnscoe'
+	}
+};
+const appeal_4 = {
+	id: 4,
+	reference: 'APP/Q8874/D/23/4293472',
+	appealStatus: [{
+		status: 'incomplete_lpa_questionnaire',
+		valid: true
+	}],
+	createdAt: new Date(2021, 12, 12),
+	addressId: 4,
+	startedAt: new Date(2022, 3, 20),
+	address: {
+		addressLine1: '180 Bloomfield Road',
+		postcode: 'BS4 3QX',
+		town: 'Bristol'
+	},
+};
+
+const appeal_5 = {
+	id: 5,
+	reference: 'APP/Q8874/D/23/1233427',
+	appealStatus: [{
+		status: 'available_for_statements',
+		valid: true
+	}],
```

Comment:
If this appeal is a Full Planning Appeal, then it would never have just this state, but it will also have a second state, for example: `[{status: 'available_for_statements', valid: true}, {status: 'incomplete_lpa_questionnaire', valid: true}]`. Could we please change this?